source/scenes/Home/components/FAQ/Before.jsx

```diff
@@ -4,35 +4,38 @@ const Before = () => (
   <section className="faq-pane">
     <div className="faq-container faq-column-container">
       <section className="faq-column">
-        <h4>What should I bring?</h4>
+        <h4>How do applications work?</h4>
         <p>
-          You should bring a student ID, a change of clothing, personal items (toiletries), a laptop
-          &amp; charger, and any required hardware for your hack. Due to power concerns, please do
-          not bring extra monitors.
+          You must apply in order to be considered for HackIllinois. Admissions will be based on a
+          weighted random lottery. Decisions will be released in January. If you’re accepted, there
+          will be a limited time to RSVP, so be on the lookout for your acceptance email!
         </p>
       </section>
       <section className="faq-column">
-        <h4>When will I find out if I can attend?</h4>
+        <h4>How do I get there?</h4>
         <p>
-          Decisions will be released one month ahead of the event in January to allow time for
-          hackers to make transportation plans. RSVP is required to check in, so be on the lookout
-          for your acceptance email!
+          We will be providing buses to certain locations across the Midwest. Last year,
```

Comment:
Given the nature of our situation with buses right now, I dont want to be too definitive on this point.